staging/src/k8s.io/noderesourcetopology-api/pkg/apis/topology/v1alpha1/types.go

```diff
@@ -0,0 +1,46 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	intstr "k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourceTopology is a specification for a Foo resource
```

Comment:
Foo

Reply:
I will update, thanks for pointing that!